app/javascript/packs/application.js

```diff
@@ -2,4 +2,29 @@ require("@rails/ujs").start()
 require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
-require("jquery")
\ No newline at end of file
+require("jquery")
+
+
+$( document ).ready(function() {
+  
+  $('#substract').on('click', () => {
+    $('#quantity')[0].value --
+    const total = parseInt($('#quantity')[0].value) * parseInt($('#servicePrice')[0].innerText.split(' ').pop())
+    $('#total')[0].innerText = ''
+    $('#total')[0].insertAdjacentText('beforeend', `$ ${total}`)
+  })
+
+  $('#add').on('click', () => {
+    $('#quantity')[0].value ++
+    const total = parseInt($('#quantity')[0].value) * parseInt($('#servicePrice')[0].innerText.split(' ').pop())
+    $('#total')[0].innerText = ''
+    $('#total')[0].insertAdjacentText('beforeend', `$ ${total}`)
```

Comment:
since this logic is the same as the above, you could extract it to a function an call it on both methods

Reply:
Thanks! Here is the commit 3f1d995